<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreloadParamsDlg</class>
 <widget class="QDialog" name="PreloadParamsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing parameters for drawings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,0,0,1,0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbRemoveInvisible">
       <property name="text">
        <string>Remove invisible entities</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="cbColorBlocks">
           <property name="text">
            <string>Paint all entities in blocks to color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leColorBlocks">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>0 = ByBlock is recommended value</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="cbColorEntities">
           <property name="text">
            <string>Paint all entities to color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leColorEntities">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>256 = ByLayer is recommended value</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="cbLWBlocks">
         <property name="text">
          <string>Set lineweight of all ents in blocks to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="listLWBlocks">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="cbLWEntities">
         <property name="text">
          <string>Set lineweight of all entities to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="listLWEntities">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="cbRename0">
         <property name="text">
          <string>Rename layer &quot;0&quot; to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leLayer0"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbRemoveXrefs">
       <property name="text">
        <string>Remove all xrefs</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbClearAnnoScales">
       <property name="text">
        <string>Clear annotation scales</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPurgeRegApps">
       <property name="text">
        <string>Purge registered applications</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPurgeAll">
       <property name="text">
        <string>Purge all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbExplodeProxy">
       <property name="text">
        <string>Explode all proxies</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbRemoveProxy">
       <property name="text">
        <string>Remove all proxies</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbAudit">
       <property name="text">
        <string>Audit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSwitchToModel">
       <property name="text">
        <string>Switch to model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSetWCS">
       <property name="text">
        <string>Set to WCS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbViewWCS">
       <property name="text">
        <string>View WCS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbZoomE">
       <property name="text">
        <string>Zoom to extents</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="cbUserCommands">
       <property name="text">
        <string>User commands</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="ptUserCommands">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbManualClose">
       <property name="text">
        <string>Manual close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbRestartAfterEach">
       <property name="text">
        <string>Restart AutoCAD after each drawing</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbDefaults">
       <property name="text">
        <string>Set to defaults</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>PreloadParamsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>844</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
